consensus/src/block_storage/sync_manager.rs

```diff
@@ -87,15 +88,16 @@ impl BlockStore {
             &retriever.network,
         )
         .await;
-        self.sync_to_highest_ordered_cert(
-            sync_info.highest_ordered_cert().clone(),
+        // TODO: Make sure syncing until highest quorum cert is okay
+        self.sync_to_highest_quorum_cert(
+            sync_info.highest_quorum_cert().clone(),
             sync_info.highest_commit_cert().clone(),
             &mut retriever,
         )
         .await?;
 
-        self.insert_quorum_cert(sync_info.highest_ordered_cert(), &mut retriever)
-            .await?;
+        // self.insert_wrapped_ledger_info(sync_info.highest_ordered_cert(), &mut retriever)
```

Comment:
we should still finalize the ordering when receive the ledger info

Reply:
Earlier, we were syncing until only highest_ordered_cert. Now, we are syncing until highest_quorum_cert. Does it still make sense to explicitly insert ordered_cert here?

Reply:
it may help finalize ordering faster, but on most happy path it probably doesn't matter. but we need to "insert" the ledger info after order vote enabled too

Reply:
remove?